docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -1,26 +1,50 @@
-# Submit a scenario, pipeline or task.
+# Submit a scenario, pipeline, or task
+
+In a Taipy application, running the Core service is required to execute jobs.
+
+To see how you can run different Taipy services, please refer to the [Running Taipy services](../../running_services/index.md) page.
+
+!!! Important "Block configuration after running Taipy Core services"
```

Comment:
- (minor) no capital for 'important'. Just homogeneity
- "Block configuration" is confusing to me (a block in a configuration?).
  What about "Preventing configuration alteration when the Taipy Core service runs"?
- "Taipy Core services"? With an 's'? What other services are there?

Reply:
I don't like block configuration either. eventually "block update" or "lock configuration". 
I would go for "Preventing configuration update when the Taipy Core service runs"

Reply:
It was updated in https://github.com/Avaiga/taipy-doc/pull/326. But again, probably I merged it too early ...
Let me know what you think